src/main.rs

```diff
@@ -25,9 +25,10 @@ async fn main() -> Result<(), Error> {
         .iter()
         .flat_map(|configuration_file| {
             debug!("Loading configuration file {}", configuration_file);
-            let configuration_result = load_file(configuration_file)
-                .and_then(|configuration| compile_configuration(&configuration))
-                .context("Error loading configuration");
+
+            // TODO how can I run my future?
+            let configuration_result = ConfLoadingFuture.load_file(configuration_file);
```

Comment:
You can't directly.
Iterator functions aren't compatbile yet with async

Reply:
I had to use a `for .. in ..` loop :'(